src/sparsezoo/analysis/Analysis.py

```diff
@@ -0,0 +1,221 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
nit: file name should be changed to avoid upper case. if the upper case was a workaround for namespace conflict with the parent directory, we can rename it to `model_analysis` or (I much less prefer) `main.py`